### PR TITLE
Update s3 bucket example pre-signed link in HTTP demo docs

### PR DESCRIPTION
**Pull request by sarenameas**
The s3 bucket and endpoint are automatically generated from the python script "presigned_url_gen.py".
The example URL in the documentation shows what could be generated. What is generated will be copied by users to their iot_config.h so it is safe to change to "aws-s3-endpoint".

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/https/README.md [hunk 1]
```diff
@@ -25,17 +25,17 @@
 ### Usage
 1. Run the presigned_url_gen.py with your s3 **bucket-name** and s3 object key **object-key**.  
    ```
-   python presigned_urls_gen.py --bucket my-bucket --key object-key
+   python presigned_urls_gen.py --bucket my_bucket --key object-key
```
Comment (gkwicker) [inline demos/https/README.md:28]: This needs to be something syntactically incorrect.  Anytime you provide shell commands, assume that someone will copy&paste them into a shell, e.g.:

```sh
    python presigned_urls_gen.py --bucket <YOUR BUCKET NAME> --key <YOUR OBJECT KEY> 
```

Comment (sarenameas) [reply demos/https/README.md:28]: Thanks

**Comment by AniruddhaKanhere** [comment]
@sarenameas, A check (`test_pkcs11_C_CloseSessionUninit`) is failing. Is it of any concern?
Just asking since you have 2 approvals and this might get merged 'as is' since the failing check is not necessary :)

**Pull request merged**